environments/production/common/iam.tf

```diff
@@ -0,0 +1,86 @@
+resource "aws_iam_user" "serverless_lambda_ci" {
+  name = "serverless-lambda-ci"
+}
+
+data "aws_kms_alias" "s3" {
+  name = "alias/s3-key"
+}
```

Comment:
This is already available to use without having to pick it out using a data source - L17 in [`s3.tf`](https://github.com/trade-tariff/trade-tariff-platform-aws-terraform/blob/main/environments/production/common/s3.tf#L17)